Modules/@babylonjs/react-native/android/src/main/java/com/babylonreactnative/EngineViewManager.java

```diff
@@ -20,6 +23,11 @@ public String getName() {
         return "EngineView";
     }
 
+    @ReactProp(name = "isTransparent")
+    public void setIsTransparent(EngineView view, Boolean isTransparent) {
```

Comment:
Does this get called for the initial prop value? I would expect it to, and if so it seems like the constructor doesn't need to create a default SurfaceView/TextureView.

Reply:
if no transparency property is set at engineView creation, this method is not called. So, the constructor needs to build the view with default parameters (transparency = false).